stdpopsim/catalog/AraTha/dfes.py

```diff
@@ -0,0 +1,52 @@
+import stdpopsim
+
+_species = stdpopsim.get_species("AraTha")
+
+###########################################################
+#
+# DFEs
+#
+###########################################################
+
+
+def _HuberDFE():
+    id = "Gamma_H18"
+    description = "Deleterious Gamma DFE"
+    long_description = """
+    Return neutral and negative MutationType()s representing an Arabidopsis DFE.
+    From Huber et al. (2018), https://doi.org/10.1038/s41467-018-05281-7.
+    Gamma parameters are based on the additive-only model for A. thaliana described in
+    Supplementary Table 4.
+    """
+    citations = [
+        stdpopsim.Citation(
+            author="Huber et al.",
+            year=2018,
+            doi="https://doi.org/10.1038/s41467-018-05281-7",
+            reasons="to be defined",  # include the dfe_model reason
```

Comment:
```suggestion
            reasons=stdpopsim.CiteReason.DFE,  # include the dfe_model reason
```

Reply:
hm, I see you copied this from `catalog/HomSap/dfes.py`; could you fix this there, also?

Reply:
got it